packages/eas-cli/src/graphql/generated.ts

```diff
@@ -3924,6 +3924,7 @@ export type IosSubmissionConfigInput = {
   ascApiKey?: InputMaybe<AscApiKeyInput>;
   ascApiKeyId?: InputMaybe<Scalars['String']['input']>;
   ascAppIdentifier: Scalars['String']['input'];
+  isVerboseFastlaneEnabled?: InputMaybe<Scalars['Boolean']['input']>;
```

Comment:
Is it auto-generated or manually edited?

Reply:
Manually

Reply:
Let's generate this automatically when WWW changes land